src/draw/DefaultDrawer.js

```diff
@@ -61,479 +90,455 @@ class DefaultDrawer {
         style: '2px solid hsl(205, 100%, 50%)',
         offset: '3px',
       },
+      linkCreation:{
+        source:null,
+        target:null,
+        creating: false
+      },
       drag: {
         state: false,
-        startX: 0,
-        startY: 0,
-      },
-      link: {
-        targetList: [],
+        target: null
       },
     };
-    /**
-     * Object to store all events.
-     * @type {{DeleteEvent: null, EditEvent: null, SelectEvent: null}}
-     */
-    this.events = {
```

Comment:
Why do you remove this ?

Reply:
was deemed unnecessary with this.events = events doing the same job, added back in